projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md

```diff
@@ -0,0 +1,29 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+description: NHSX PhD Internship - Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+permalink: /Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/
+---
+
+# Graph 
+
+**Keywords:** Graph Structures, Data Representation, Hypergraphs
+
+**Need:** The collections of linked health datasets into a single research environment creates a number of large opportunities for insight and analysis.  Whilst anonymised extracts from these environments have great value, there is an even larger opportunity to develop methods of deploying modelling to the data remotely.  [The SAIL Databank - The Secure Anonymised Information Linkage Databank](https://saildatabank.com/) provides anonymised person-based data for research powered by the Secure e-Research Platform.  
+
+Recent work using graph models to build simpler knowledge discovery systems opens up potential for increased prediction accuracies, reduced pre-processing burden and the application of higher complexities models to our data.  These models have been shown to effectively handle messy data and to learn representation of key factors from the data directly (rather than choosing a set of predictor variables).
+
+**Current Knowledge/Examples & Possible Techniques/Approaches:** The recent paper entitled [Ranking Sets of Morbidities using Hypergraph Centrality](https://www.sciencedirect.com/science/article/pii/S1532046421002458) and accompanying [github repo] (https://github.com/jim-rafferty/multimorbidity_hypergraphs) demonstrates applying a hypergraph analysis to a comorbidity question.  Where possible, this project would seek to develop upon this work looking at including further complexity through adding demographics, direction into the graphs, or time evolution.  A likely first step to this would be a comparison of hypergraphs application versus state-models and more traditional clustering.
```

Comment:
`and more traditional clustering.` -> `and more traditional clustering approaches.`